rules/sinks/storages/hibernate/java.yaml

```diff
@@ -1,15 +1,13 @@
-
-
 # Sink Rules for storage database Hibernate - https://hibernate.org/
 
 sinks:
-
   - id: Storages.Hibernate.Read
     name: Hibernate(Read)
     domains:
       - hibernate.org
-    patterns: 
-      - "(?i)((org[.]hibernate[.]Session[.]((byId|byMultipleIds|byNaturalId|bySimpleNaturalId|get|getEntityName|load|refresh)[:].*))|(org[.]hibernate[.]query[.]Query.*((getResultList|getResultStream|getSingleResult|list||stream|uniqueResult|uniqueResultOptional)[:].*)))"
+    patterns:
+      - "(?i)(org[.]hibernate[.]Session[.])((byId|byMultipleIds|byNaturalId|bySimpleNaturalId|get|getEntityName|load|refresh)[:])(.*)"
+      - "(?i)(org[.]hibernate[.])(query[.])? (Query.*)(getResultList|getResultStream|getSingleResult|list|stream|uniqueResult|uniqueResultOptional)[:](.*)"
```

Comment:
@gaurav-gogia Is this optional `(org[.]hibernate[.])(query[.])` as you have placed a `?` after this

Reply:
Yes, that's optional.

In some cases we see `hibernate.query.Query` and in some cases we have seen `hibernate.Query`